src/async_impl/request.rs

```diff
@@ -179,6 +180,47 @@ impl RequestBuilder {
         self
     }
 
+    /// Sends a multipart/form-data body.
+    ///
+    /// ```
+    /// # extern crate futures;
+    /// # extern crate reqwest;
+    ///
+    /// # use reqwest::Error;
+    /// # use futures::future::Future;
+    ///
+    /// # fn run() -> Result<(), Error> {
+    /// let client = reqwest::async::Client::new();
+    /// let form = reqwest::async::multipart::Form::new()
+    ///     .text("key3", "value3")
+    ///     .text("file", "/path/to/field");
```

Comment:
This threw me off at first, it's not actually going to include a file at `/path/to/field`. Perhaps it shouldn't look like adding a file so people don't assume wrongly?

Reply:
Yeah that's a good idea.